rh-shield-operator/config/manifests/bases/rh-shield-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: SeamlessUpgrades
+    categories: Security, Monitoring
+    description: |
+      The Sysdig Shield Operator provides a way to deploy Sysdig Shield components on an OpenShift cluster.
+    repository: https://github.com/sysdiglabs/charts
+    support: https://sysdig.com
+  name: rh-shield-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: |
+    [Sysdig](https://www.sysdig.com/) is a unified platform for container and
+    microservices monitoring, troubleshooting, security, and forensics. Sysdig
+    platform has been built on top of the [Sysdig tool](https://sysdig.com/opensource/sysdig/) and
+    [Sysdig Inspect](https://sysdig.com/blog/sysdig-inspect/) open-source technologies.
+    This operator installs the Sysdig Cluster and Host Shields for
+    [Sysdig Monitor](https://sysdig.com/product/monitor/) and
+    [Sysdig Secure](https://sysdig.com/product/secure/).
+  displayName: Sysdig Shield Operator
+  icon:
+  - base64data: "PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB2aWV3Qm94PSIwIDAgMjE3IDIwOCI"
```

Comment:
This looks like an empty SVG image (fine for now, we may open a different task in order to use the proper logo here)

Reply:
Nice catch! I've updated the content.